test/sequential/test-perf-hooks.js

```diff
@@ -1,84 +1,165 @@
 'use strict';
 
 const common = require('../common');
-const assert = require('assert');
 const { performance } = require('perf_hooks');
+// Get the start time as soon as possible.
+const testStartTime = performance.now();
+const assert = require('assert');
+const { writeSync } = require('fs');
 
-if (!common.isMainThread)
-  common.skip('bootstrapping workers works differently');
+// Use writeSync to stdout to avoid disturbing the loop.
+function log(str) {
+  writeSync(1, str + '\n');
+}
 
 assert(performance);
 assert(performance.nodeTiming);
 assert.strictEqual(typeof performance.timeOrigin, 'number');
+
+assert(testStartTime > 0, `${testStartTime} <= 0`);
 // Use a fairly large epsilon value, since we can only guarantee that the node
 // process started up in 15 seconds.
-assert(Math.abs(performance.timeOrigin - Date.now()) < 15000);
+assert(testStartTime < 15000, `${testStartTime} >= 15000`);
```

Comment:
nit
```suggestion
assert(testStartTime < 15000, `${testStartTime} ≥ 15000`);
```

Reply:
I think in general we use >= in this codebase instead?